README.md

```diff
@@ -175,46 +184,50 @@ After downloading/installing the prerequisites, ensure that you can execute the
 3. On Windows: `where sh`
 ```
 
-To run Zowe CLI system tests, you need a configured properties file with proper system information present.
+To run Zowe CLI system tests, you need a configured properties file populated with proper system information.
 
-A dummy properties file is present in the `__tests__/__resources__/properties folder`, `default_properties.yaml`. Using this file as a template, you should create a `custom_properties.yaml` file within the same directory. Git is configured to ignore all properties files in the properties folder, except for the `default_properties.yaml` file. If the `custom_properties.yaml` file cannot be found or loaded, an error with relevant details will be thrown when attempting to run tests.
+A dummy properties file is available in the `default_properties.yaml` file **[is this correct? I couldn't find it in the repo]** in the `__tests__/__resources__/properties` folder. Using this file as a template, you should create a `custom_properties.yaml` file within the same directory. Git is configured to ignore all properties files in the properties folder, except for the `default_properties.yaml` **[is this correct? I couldn't find it in the repo]** file. If the `custom_properties.yaml` file cannot be found or loaded, an error with relevant details displays when attempting to run tests.
```

Comment:
@adam-wolfe @ATorrise - please respond

Reply:
The default_properties.yaml file is here: https://github.com/zowe/zowe-cli/blob/master/__tests__/__resources__/properties/default_properties.yaml

Also, the gitignore file does contain:
```
/__tests__/__resources__/properties/**
!/__tests__/__resources__/properties/default_properties.yaml
```
Meaning that all files in `/__tests__/__resources__/properties/` are ignored except for `default_properties.yaml`

So, I think this is correct.

Reply:
@adam-wolfe - Was looking at the wrong repo, thank you for clarifying.